.github/workflows/test.yml

```
@@ -184,12 +184,25 @@ jobs:
       #        ]
       #       }
 
-      - name: Acceptance tests
+      - name: "Acceptance tests"
+        if: ${{ ! ( matrix.flavor.target == 'windows' && github.event.pull_request.draft  ) }}
+        uses: robherley/go-test-action@v0
+        with:
+          testArguments: -v -coverpkg=./... -timeout 40m -coverprofile=coverage.out.tmp
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)